### ISSUE TITLE
Security scanner reports CVEs in modules. Plans for updating them ?

### ISSUE THREAD

**Issue by rafariossaa** [GoogleContainerTools/kaniko#2969]
Hi, 
  When running trivy scanner on kaniko 1.19.2, it reports several CVEs modules used by kaniko. Is there any plan to upgrade them ?

```
kaniko/docker-credential-acr-env (gobinary)

Total: 8 (UNKNOWN: 0, LOW: 0, MEDIUM: 2, HIGH: 6, CRITICAL: 0)
```

```
kaniko/docker-credential-gcr (gobinary)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 0)

```

```
kaniko/executor (gobinary)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

```

Thanks forehand

**Comment by aaron-prindle** [comment]
@rafariossaa can you try scanning the latest release of kaniko - `v1.20.0`.  I believe have already been resolved IIUC:
```
aprindle@aprindle-ssd ~/kaniko  [nightly-vuln-scan-v2]grype gcr.io/kaniko-project/executor:latest
 ✔ Vulnerability DB        [updated]
New version of grype is available: 0.74.3 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [215 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

**Comment by aaron-prindle** [comment]
I believe this was resolved at this time, this issue tracks additional CVEs flagged at this time:
https://github.com/GoogleContainerTools/kaniko/issues/2992
